packages/client/hmi-client/src/components/responsive-matrix/matrix.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<main class="matrix-container" ref="matrixContainer">
+	<main class="matrix-container" ref="matrixContainer" :class="{ cameraCursor: isCameraMode }">
```

Comment:
we should probably have one class property attached to this element rather than multiple. in addition, ive been using kebab case for class properties throughout the responsive matrix stuff, if we want to change that we should probably do it everywhere, but until that happens probably a good idea to keep this consistent and stick with kebab case.

Reply:
What's the rational for only having one class attached to this element?

Reply:
on some more looking it appears that having both a `class` and `:class` on the same element is normal, i personally dont really like that but if thats how its supposed to be done its fine. i would prefer if we do something like the following for the class name.

```
<main ref="matrixContainer" class="matrix-container" :class="{ 'camera-cursor': isCameraMode }">